test/javascript/explore-data/search-matrix-spec.js

```diff
@@ -131,17 +146,57 @@ describe("Searching a matrix (table) of data...", function() {
 		});
 
 		it("should survive incomplete input", function() {
-			expect(visualizeMatrix([["t","o"],["b"],["l","o","n","g"]])).toEqual("to\nb\nlong");
+			expect(visualizeMatrix([
+				["t", "o"],
+				["b"],
+				["l", "o", "n", "g"]
+			])).toEqual("to\nb\nlong");
 		});
 
 	});
 
-	it("should run in finite time", function() {
-		expect(true).toBeTruthy();
-	});
+	describe("performance", function() {
+
+		xit("should run in finite time", function() {
+			try {
```

Comment:
xit turns off the test. in this case the test was making it slow to load the tests, so it works but we turned it off so that it didnt run when we were working.